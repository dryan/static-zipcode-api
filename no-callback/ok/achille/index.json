{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["74720"], "name": "Achille", "counties": [{"fips": "013", "name": "Bryan"}]}