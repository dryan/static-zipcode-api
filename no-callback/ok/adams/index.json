{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["73901"], "name": "Adams", "counties": [{"fips": "139", "name": "Texas"}]}