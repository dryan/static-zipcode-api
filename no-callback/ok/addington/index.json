{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["73520"], "name": "Addington", "counties": [{"fips": "067", "name": "Jefferson"}]}