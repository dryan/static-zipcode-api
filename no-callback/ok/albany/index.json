{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["74721"], "name": "Albany", "counties": [{"fips": "013", "name": "Bryan"}]}