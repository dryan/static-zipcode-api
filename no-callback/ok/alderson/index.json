{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["74522"], "name": "Alderson", "counties": [{"fips": "121", "name": "Pittsburg"}]}