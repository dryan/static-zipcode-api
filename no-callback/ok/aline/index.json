{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["73716"], "name": "Aline", "counties": [{"fips": "093", "name": "Major"}, {"fips": "003", "name": "Alfalfa"}, {"fips": "151", "name": "Woods"}]}