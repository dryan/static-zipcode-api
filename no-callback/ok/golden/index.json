{"region": {"fips": "40", "name": "Oklahoma", "abbr": "OK"}, "postal_codes": ["74737"], "name": "Golden", "counties": [{"fips": "089", "name": "McCurtain"}]}